tests/integration/tests/test_standalone_agent_integration.py

```diff
@@ -83,6 +83,7 @@ def test_elastic_index_exists(elastic_client, match_type):
 @pytest.mark.post_merge_agent
 @pytest.mark.order(4)
 @pytest.mark.dependency(depends=["test_agent_pods_running"])
+@pytest.mark.skip(reason="Broken")
```

Comment:
can you please provide a better description?

Reply:
Opened an issue and linked it here https://github.com/elastic/cloudbeat/issues/506 as well as mentioned this ticket & pr